docs/cloudfoundry/cf-per-route-virtuals.rst

```diff
@@ -84,54 +90,64 @@ Register the BIG-IP Controller as a Service Broker
    You'll need to provide the following information:
 
    - a name for the Service Broker;
-   - a username and password for a user with permission to interact with the Service Broker API (sames as in the :code:`bigip.status` section of the Application Manifest);
+   - a username and password for a user that can interact with the Controller's API (this can be the same user provided in the :code:`bigip.status` section of the Application Manifest);
```

Comment:
There is only one user and that's the one defined in status, so it's required to use that info to run as a sb